packages/replayio/src/utils/recordings/upload/uploadCrashData.ts

```diff
@@ -14,7 +14,7 @@ export async function uploadCrashedData(client: ProtocolClient, recording: Local
     recordingId: recording.id,
   });
 
-  await Promise.all(crashData.map(async data => reportCrash(client, data)));
+  await Promise.all(crashData.map(async data => reportCrash(client, { data })));
```

Comment:
The [docs](https://static.replay.io/protocol/tot/Internal/#method-reportCrash) are not very clear, but this was wrong before.